src/page-modules/contact/validation/commonFieldValidator.ts

```diff
@@ -6,17 +6,39 @@ export const commonFieldValidator = (context: any) => {
   const inputFieldRules = [
     {
       field: 'firstName',
-      validCondition: context.firstName,
+      validCondition:
+        context.firstName ||
+        context.formType === 'driver' ||
+        context.formType === 'transportation' ||
+        context.formType === 'delay' ||
+        context.formType === 'stop' ||
+        context.formType === 'serviceOffering' ||
+        context.formType === 'injury',
```

Comment:
Could this be extracted to a separate function? Something like "isContactInfoOptional"